doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -139,5 +140,5 @@ Write-Verbose "Build Doc Content"
 Write-Verbose "Copy over site Logo"
 Copy-Item "${DocGenDir}/assets/logo.svg" -Destination "${DocOutHtmlDir}" -Recurse -Force
 
-Write-Verbose "Set variable for publish pipeline step"
-echo "##vso[task.setvariable variable=PublishTargetPath]${DocOutHtmlDir}"
\ No newline at end of file
+Write-Verbose "Compress and copy HTML into the staging Area"
+Compress-Archive -Path "${DocOutHtmlDir}/*" -DestinationPath "${ArtifactStagingDirectory}/${ArtifactName}/${ArtifactName}.docs.zip" -CompressionLevel Fastest
```

Comment:
Is there a reason we zipping these files? We could also just put them in a folder that would make this step and the extraction step unnecessary.

Reply:
No particular reason, other that it make downloading from the UI a bit faster because otherwise DevOps first zips it before downloading it.

Reply:
Ok in that case I'd suggest we just avoid this round tripping and just put the files directly in an artifact folder.

Reply:
Once you finish your other work to move the language specific implementation of this into the .NET repo I would still like us to change this from being a zip file to just a folder in the artifacts.